app/templates/users/includes/tables.html

```diff
@@ -80,10 +80,10 @@ <h2 class="heading-medium">{{ heading_prefix }} app data sources</h2>
           </td>
           <td class="align-right">
             {% if users3bucket.is_admin %}
-              <a class="button button-secondary" href="{{ url_for('buckets.details', {id: users3bucket.id}) }}">Manage bucket access</a>
+              <a class="button button-secondary" href="{{ url_for('buckets.details', {id: users3bucket.id}) }}">Manage data source access</a>
```

Comment:
Are we replacing all current instances of the `bucket` word with `app data source`?

Reply:
That's the broad idea, at least in copy that the user sees.